utilities/options/options_util_test.cc

```diff
@@ -386,11 +377,12 @@ TEST_F(OptionsUtilTest, LatestOptionsNotFound) {
   ASSERT_TRUE(s.IsNotFound());
   ASSERT_TRUE(s.IsPathNotFound());
 
-  s = LoadLatestOptions(dbname_, options.env, &options, &cf_descs);
-  ASSERT_TRUE(s.IsNotFound());
-  ASSERT_TRUE(s.IsPathNotFound());
+  //  s = LoadLatestOptions(dbname_, options.env, &options, &cf_descs);
+  //  ASSERT_TRUE(s.IsNotFound());
+  //  ASSERT_TRUE(s.IsPathNotFound());
```

Comment:
This is testing one of the deprecated overloads, right? If so, we can just remove it

Reply:
Yes, thank you for catching this. I forgot to remove it.